bin/setup

```diff
@@ -2,6 +2,25 @@
 
 set -e
 
+echo "[setup]: Installing required languages"
```

Comment:
Can you add some more `echo`s on this? On the other steps of the setup, for it to be more consistent.

Reply:
Done